src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CngKeyLite.cs

```diff
@@ -94,7 +134,7 @@ internal static byte[] ExportKeyBlob(SafeNCryptKeyHandle keyHandle, string blobT
                 IntPtr.Zero,
                 blobType,
                 IntPtr.Zero,
-                buffer,
+                ref buffer[0],
```

Comment:
numBytesNeeded it guaranteed to be > 0?

Reply:
It doesn't really make sense in context that a zero-byte thing could be exported; but I added an if-zero-return-empty check anyways.